package.json

```diff
@@ -1,15 +1,13 @@
 {
     "name": "dkehrig.show-whitespace",
     "title": "Show Whitespace",
-    "description": "Visualizes whitespace similar to how Sublime Text does it",
+    "description": "Visualize whitespace (both spaces and tabs) in the same style as Sublime Text. Now with configurable whitespace colors!",
```

Comment:
I don't know about others, but I often use the description to highlight very notable changes in the new and/or previous few versions.